### DISCUSSION TITLE
there is a bug for Fabric API

### DISCUSSION THREAD

**Discussion by Prinxeyyz** [FabricMC/fabric-api#3443] [Mod Dev Support]
### Description

when  I launch into minecraft all I see are 0's and squares please fix the version, the version is   1.20.1 please fix because I want to use Fabric API for myself 
<img width="1440" alt="Screen Shot 2023-11-26 at 9 12 17 AM" src="https://github.com/FabricMC/fabric/assets/152085274/791bca84-e44a-48e6-95f7-b3279c6ea80c">

**Comment by rilakuma123** [comment]
Got any more information? What is your mod trying to do and how?

**Comment by Linguardium** [comment]
this looks like someone trying to use a janky mod like optifine to me. if using optifine, dont.

**Comment by TheBestAstroNOT** [comment]
It seems like the language locale files are missing. I.e the files which actually provide the text to display according to your system language. Most likely not related to fabric api. Maybe some other mod nuked em or u deleted em by mistake. Try making a new installation with the mods or uninstall each mod one by one to check which one is problematic.
